### ISSUE TITLE
Add notes to keywords (e.g. Monthly number of searches)

### ISSUE THREAD

**Issue by luckyshot** [serphacker/serposcope#106]
It would be useful to add a small note on each keyword, specifically the Monthly number of searches, so we can have better insights on each keyword's importance directly from Serposcope.

**Comment by serphacker** [comment]
Not a bad idea, I already want to add search volume to keywords, but maybe I'll think to something more generic so we can add any kind of metadata to keywords.

**Comment by luckyshot** [comment]
Yes! Calculating the search volume programmatically would be great and save a lot of time from manually having to update each one, although maybe adding the metadata field is probably much quicker to implement.

**Comment by tontognolli** [comment]
The keyword data differs per country, so there should be some settings available for what country or region, so that the data is trustworthy. Eg. Certain keywords give for the US a very high result where the Netherlands give a very low result on the same keyword. Hopefully you take this under consideration?
best regards, ton
